src/open_inwoner/accounts/views/auth.py

```diff
@@ -1,16 +1,31 @@
-from django.contrib.auth import update_session_auth_hash
+from django.contrib.auth.mixins import UserPassesTestMixin
 from django.contrib.auth.views import (
     PasswordChangeView,
     PasswordResetConfirmView,
     PasswordResetView,
 )
-from django.http import HttpResponseRedirect
+from django.shortcuts import redirect
+from django.urls import reverse
 from django.utils.translation import gettext as _
 
 from open_inwoner.utils.views import LogMixin
 
+from ..choices import LoginTypeChoices
+from ..forms import CustomPasswordResetForm
+
+
+class LogPasswordChangeView(UserPassesTestMixin, LogMixin, PasswordChangeView):
+    def test_func(self):
+        if self.request.user.is_authenticated:
+            return self.request.user.login_type == LoginTypeChoices.default
+        return False
+
+    def handle_no_permission(self):
+        if self.request.user.is_authenticated:
+            return redirect(reverse("root"))
```

Comment:
Maybe include django message that this user can't change the password here?

Reply:
Is this needed? I am asking because this is only a secondary check if the digid user types explicitly the url for the password-change view. By default the user cannot see this button (it is not rendered). We have implemented the same  case in "Mijn aanvragen" page.

Reply:
Lets remove the function.